### ISSUE TITLE
💡 Edge Add-ons weekly download

### ISSUE THREAD

**Issue by johannchopin** [badges/shields#7349]
:clipboard: **Description**

Add a new badge for the users amount of an **Edge Add-ons** just like the one for the **Chrome Web Store** `/chrome-web-store/users/:storeid`:
![image](https://user-images.githubusercontent.com/31794680/144748664-68e662b9-b711-4ffe-a7a1-f902e75e4570.png)

:link: **Data**

Microsoft seems to have opened a new REST API for the Add-ons. You can have the amount of daily users by calling:

```
https://partner.microsoft.com/analytics/api/vnext/edgeaddons/weeklyActiveInstalls?applicationId=APP_ID&orderBy=date%20asc&groupBy=date&filter=userCount%20gt%20%270%27&startDate=START_DATE
```

Example:
```
https://partner.microsoft.com/analytics/api/vnext/edgeaddons/weeklyActiveInstalls?applicationId=0RDCKBGFN59T&orderBy=date%20asc&groupBy=date&filter=userCount%20gt%20%270%27&startDate=11/5/2021
```

You will get back a json like:

```json
{
  "Value": [
    {
      "date": "2021-11-07",
      "applicationId": "0RDCKBGFN59T",
      "userCount": 676
    },
    {
      "date": "2021-11-14",
      "applicationId": "0RDCKBGFN59T",
      "userCount": 651
    },
    {
      "date": "2021-11-21",
      "applicationId": "0RDCKBGFN59T",
      "userCount": 622
    },
    {
      "date": "2021-11-28",
      "applicationId": "0RDCKBGFN59T",
      "userCount": 604
    }
  ],
  "TotalCount": 4
}
```

It is then possible to calculate the average and send it back in the badge. Sadly I didn't really find an official documentation for this one.

:microphone: **Motivation**

It's really important for browser extensions to be easily able to show in a README how much users they already have. It's possible for Chrome and firefox but not edge since there was no API. Now that one is available, it would be nice to implement it. I could try to implement this new badge by my own since it's simply a copy of the **Chrome Web Store** one.

Thanks again for your work on this project 🙏 I got you some ☕s on opencollective 😉

**Comment by chris48s** [comment]
Hi. Thanks for supporting the project.

I tried calling https://partner.microsoft.com/analytics/api/vnext/edgeaddons/weeklyActiveInstalls?applicationId=0RDCKBGFN59T&orderBy=date%20asc&groupBy=date&filter=userCount%20gt%20%270%27&startDate=11/5/2021 and it gives me a `401 Unauthorized`. Does this API require authentication? Are there any docs for it?

**Comment by johannchopin** [comment]
@chris48s Yes sorry for not having tried in incognito mode. There is no bearer token in the request header but the complete cookie with a session token. I tried again to search for any doc but the one from the microsoft partner center https://docs.microsoft.com/en-us/partner-center/develop/partner-center-rest-resources doesn't seems to have any documentation related to the edge-addons 😢 I guess we can't really go forward if there is no official doc.

**Comment by PyvesB** [comment]
Let's consolidate the discussion for Edge Add-on badges in #4690. Thanks for opening and adding the details! 😉
